src/release/breaking-changes/shortcut-key-event-migration.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Migrate ShortcutActivator and ShortcutManager to KeyEvent system.
```

Comment:
nit: I believe we usually don't put a `.` at the end of these titles? But please double-check

Reply:
Done